docs/GenericsManifesto.md

```diff
@@ -757,8 +757,8 @@ if e1 == e2 { ... } // error: e1 and e2 don't necessarily have the same dynamic
 One explicit way to allow such operations in a type-safe manner is to introduce an "open existential" operation of some sort, which extracts and gives a name to the dynamic type stored inside an existential. For example:
 
 ```Swift
-if let storedInE1 = e1 openas T {     // T is a the type of storedInE1, a copy of the value stored in e1
-  if let storedInE2 = e2 as? T {      // is e2 also a T?
+if let storedInE1 = e1 openas T { // T is the type of storedInE1, a copy of the value stored in e1
+  if let storedInE2 = e2 as? T {  // does e2 have type T? if so, copy its value to storedInE2
```

Comment:
Nit: capitalization

Reply:
I added capitalization in https://github.com/apple/swift/pull/17453/commits/2dc279fa47cd53ef371f1aead65c9ddc719f9735, is this what you had in mind?
I originally didn't capitalize comments for consistency with the rest of the doc.